process_report/tests/unit/invoices/test_pi_specific_invoice.py

```diff
@@ -144,3 +164,26 @@ def test_export_pi(self, mock_subprocess_run, mock_path_exists, mock_filter_cols
                     "--no-pdf-header-footer",
                 ]
                 self.assertTrue(answer_arglist == chrome_arglist[0][:-1])
+
+    def test_process_no_warnings(self):
+        """Test that no warnings are raised during invoice processing"""
+        test_invoice = self._get_test_invoice(
+            ["PI1", "PI1", "PI2", "PI2"],
+            ["BU", "BU", "HU", "HU"],
+            [100, 200, 300, 400],
+            group_name=[None, "G1", None, None],
+        )
+
+        pi_inv = test_utils.new_pi_specific_invoice(data=test_invoice)
+
+        # Capture all warnings
+        with warnings.catch_warnings(record=True) as w:
+            warnings.simplefilter("always")
+            pi_inv.process()
+
+            # Assert no warnings were raised
```

Comment:
These 2 comments are not necessary. As mentioned [before](https://github.com/CCI-MOC/invoicing/pull/198#discussion_r2110368427), we usually only wrote comments to explain the WHY, unless the code is really complicated and the intention not clear. If the code is not clear in the first place, the problem might be the code, not a lack of comments :)